docker/lntop/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.12-alpine as builder
+
+# install build dependencies
+RUN apk add --no-cache --update git gcc musl-dev
+
+ARG LNTOP_SRC_PATH
+
+WORKDIR /root/_build
+
+# we want to populate the module cache based on the go.{mod,sum} files.
+COPY "$LNTOP_SRC_PATH/go.mod" .
+COPY "$LNTOP_SRC_PATH/go.sum" .
+
+# pre-cache deps
+# see https://container-solutions.com/faster-builds-in-docker-with-go-1-11/
+RUN go mod download
```

Comment:
Vendor contains all the dep. I forgot to mention it in README.
I think you can remove this line, if we use `go install -mod=vendor ./...`